front/src/modules/settings/profile/components/DeleteWorkspace.tsx

```diff
@@ -0,0 +1,128 @@
+import { Dispatch, FC, SetStateAction, useEffect, useState } from 'react';
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+import debounce from 'lodash.debounce';
+import { useRecoilValue } from 'recoil';
+
+import { StyledContainer } from '@/auth/components/Modal';
+import { currentUserState } from '@/auth/states/currentUserState';
+import { Button } from '@/ui/button/components/Button';
+import { TextInput } from '@/ui/input/components/TextInput';
+import { Modal } from '@/ui/modal/components/Modal';
+import { SubSectionTitle } from '@/ui/title/components/SubSectionTitle';
+
+const CenteredButton = styled(Button)`
```

Comment:
By convention, we prefix all emotion styled component by Styled to make easier to understand it's just styling when we use it